students/drageon/users/views.py

```diff
@@ -50,13 +51,17 @@ def post(self, request):
         data = json.loads(request.body)
 
         try : 
-            email         = data["email"] 
-            password      = data["password"]
-            
-            if not User.objects.filter(email=email, password=password).exists() :
+            email    = data["email"] 
+            password = data["password"]
+            hashed_Password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Comment:
대문자가 들어가 있네요!! 수정해주세요~

```suggestion
            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Reply:
수정 완료 하였습니다.